srcpkgs/CalculiX-cgx/files/README_LICENSES

```diff
@@ -0,0 +1,5 @@
+cgx program is released under GNU GPL v2 or later (except for a few functions, see CGX_LICENSE)
+
+To be compiled, this program makes use of other programs released under different licenses:
+* glut is released under a custom license (see GLUT_LICENSE)
```

Comment:
the glut licence's status as free software is dubious. maybe this should be `repository=nonfree`, not sure

Reply:
Reading the GLUT licence, it seems to me that it's essentially a MIT-like permissive licence

Reply:
ok that seems right to me